synapse/appservice/scheduler.py

```diff
@@ -158,9 +173,16 @@ def __init__(self, clock, store, as_api):
         # for UTs
         self.RECOVERER_CLASS = _Recoverer
 
-    async def send(self, service, events):
+    async def send(
+        self,
+        service: ApplicationService,
+        events: List[EventBase],
+        ephemeral: Optional[JsonDict] = None,
```

Comment:
Isn't this a `Optional[List[JsonDict]]`? `enqueue_ephemeral` seems to take a `List[JsonDict]` which I think then pops back out here via `_send_request`?

Reply:
I decided to make it not optional, because we use an empty list in most places.